sagemaker-lineage/sagemaker-lineage.ipynb

```diff
@@ -268,6 +270,7 @@
    "source": [
     "## Caveats\n",
     "\n",
+    "* CreateArtifact API may only be called from a SageMaker context such as Studio or Training Job.\n",
```

Comment:
Change to: `CreateArtifact` is a SageMaker API call that may only be called from a SageMaker context i.e., from a SageMaker training job.

Reply:
updated